demos/python_demos/human_pose_estimation_3d_demo/modules/inference_engine_openvino.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import numpy as np
+from openvino.inference_engine import IENetwork, IECore
+
+
+class InferenceEngineOpenVINO:
+    def __init__(self, net_model_xml_path, device, stride):
+        self.device = device
+        self.stride = stride
+
+        net_model_bin_path = os.path.splitext(net_model_xml_path)[0] + '.bin'
+        self.net = IENetwork(model=net_model_xml_path, weights=net_model_bin_path)
+        required_input_key = {'data'}
+        assert required_input_key == set(self.net.inputs.keys()), \
+            'Demo supports only topologies with the following input key: {}'.format(', '.join(required_input_key))
+        required_output_keys = {'features', 'heatmaps', 'pafs'}
+        assert required_output_keys.issubset(self.net.outputs.keys()), \
+            'Demo supports only topologies with the following output keys: {}'.format(', '.join(required_output_keys))
+
+        self.ie = IECore()
+        self.exec_net = self.ie.load_network(network=self.net, num_requests=1, device_name=device)
+
+    def infer(self, img):
+        img = img[:, 0:img.shape[1] - (img.shape[1] % self.stride)]
```

Comment:
Cropping in that that way does not seem to make any sense because this is what net's infer will automatically do